docker/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM ubuntu:20.04
+
+# configure the "generator" user
+RUN groupadd generator && \
```

Comment:
Codacy found an issue: [Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`](https://app.codacy.com/manual/ndelvalle/generator-api/pullRequest?prid=6300885)

Reply:
specifying version for common packages will be too much. We can ignore this.